packages/schematics/angular/utility/find-module.ts

```diff
@@ -26,8 +27,7 @@ export const ROUTING_MODULE_EXT = '-routing.module.ts';
  * Find the module referred by a set of options passed to the schematics.
  */
 export function findModuleFromOptions(host: Tree, options: ModuleOptions): Path | undefined {
-  // eslint-disable-next-line no-prototype-builtins
-  if (options.hasOwnProperty('skipImport') && options.skipImport) {
+  if (options.standalone || options.skipImport) {
```

Comment:
It might make sense to get rid of the `skipImport` option at some point.

Reply:
I was considering that, but it could cause unintended consequences if the behavior of `skipImport` changes.